src/Microsoft.IdentityModel.Tokens/Validation/Validators.Algorithm.cs

```diff
@@ -42,5 +43,65 @@ public static void ValidateAlgorithm(string algorithm, SecurityKey securityKey,
                 });
             }
         }
+
+        /// <summary>
+        /// Validates a given algorithm for a <see cref="SecurityKey"/>.
+        /// </summary>
+        /// <param name="algorithm">The algorithm to be validated.</param>
+        /// <param name="securityKey">The <see cref="SecurityKey"/> that signed the <see cref="SecurityToken"/>.</param>
+        /// <param name="securityToken">The <see cref="SecurityToken"/> being validated.</param>
+        /// <param name="validationParameters"><see cref="TokenValidationParameters"/> required for validation.</param>
+        /// <param name="callContext"></param>
+#pragma warning disable CA1801 // TODO: remove pragma disable once callContext is used for logging
+        internal static AlgorithmValidationResult ValidateAlgorithm(string algorithm, SecurityKey securityKey, SecurityToken securityToken, TokenValidationParameters validationParameters, CallContext callContext)
```

Comment:
Please align the parameters one per line:

```csharp
internal static AlgorithmValidationResult ValidateAlgorithm(  
    string algorithm,  
    SecurityKey securityKey,   
    SecurityToken securityToken,   
    TokenValidationParameters validationParameters,   
    CallContext callContext)
```

Reply:
Also we should wrap this in a #nullable block and mark `CallContext` as nullable to make it optional.

Reply:
Updated, Thanks @iNinja !